form.yml.erb

```diff
@@ -140,28 +154,32 @@ attributes:
           data-min-ppn-pitzer: 1,
           data-max-ppn-pitzer: 48,
           data-option-for-owens: true,
-          data-option-for-pitzer: true
+          data-option-for-pitzer: true,
+          data-option-for-kubernetes: false
```

Comment:
same as any cpu, why not any gpu for k8s?

Reply:
See other comment amount mounts and Lmod and hardware compatibility.